### ISSUE TITLE
Perf: highlighted slot style on ctrl_item_slot_ex

### ISSUE THREAD

**Issue by LeeWannacott** [peter-kish/gloot#199]
This is running whilst moving the mouse around (InputEventMouseMotion) not even doing anything with the actual slot.

The best part is, I don't have a slot_highlighted_style set on any of my ItemSlots.

I wonder if there is a better way to do this?; like using the signal for mouse_entered, mouse_exited?

Cheers.

Line 51 in ctrl_item_slot_ex.gd:
![image](https://github.com/peter-kish/gloot/assets/49783296/4c09d046-9c22-4eaa-81ad-d703c6135f6b)

**Comment by peter-kish** [comment]
> I wonder if there is a better way to do this?; like using the signal for mouse_entered/focus_entered, mouse_exited/focues_exited?

Does this cause any performance issues for you? But regardless, I agree, updating the control every time the mouse is moved is a bit too aggressive. There's definitely a better way.

**Comment by LeeWannacott** [comment]
> Does this cause any performance issues for you?

Not really, but I have a half decent CPU (5900x), typically you don't make games targeted at your own hardware.

**Comment by peter-kish** [comment]
Should be fixed in [v2.4.9](https://github.com/peter-kish/gloot/releases/tag/v2.4.9).
